### PR TITLE
Implement cover entity

### LINKED ISSUE #72: Add BlindActuator as cover entity

### PR DESCRIPTION
**Pull request by derjoerg**
closes #72

### CODE REVIEW

File: custom_components/abbfreeathome_ci/strings.json [hunk 1]
```diff
@@ -72,6 +72,20 @@
...
+      "awning_actuator": {
+        "name": "{channel_name} Awning"
+      },
+      "blind_actuator": {
+        "name": "{channel_name} Blind"
+      },
+      "shutter_actuator": {
+        "name": "{channel_name} Shutter"
+      }
+    },
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/strings.json:88]: Are the translations required for this one? Are the channel_name’s not unique? I’d would have thought they were but I could be wrong.

If I called my blinds in Free@Home “Living Room Blinds” I imagine they’ll come through in Home Assistant as “Living Room Blinds Blind”.

Comment (kingsleyadam) [reply custom_components/abbfreeathome_ci/strings.json:88]: Also, to add, looking at the example here: https://github.com/kingsleyadam/local-abbfreeathome/pull/79

I can see the channel name is unique from the device name. In this case I’d say the translations can be removed and we just use the channel name.

**Comment by derjoerg** [comment]
Please hold on with the review. Something with the device-classes in HA seems to be odd. I have to investigate a bit (nothing technical, just the look-and-feel).

**Comment by derjoerg** [comment]
My last change looks a bit strange, but:
- a BlindActuator in F@H is - from a HA device_class - in HA a shutter
- a ShutterActuator in F@H is - from a HA device_class - in HA a blind

File: custom_components/abbfreeathome_ci/cover.py [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import CONF_SERIAL, DOMAIN
+
+SELECT_DESCRIPTIONS = {
+    "AtticWindowActuator": {
+        "device_class": AtticWindowActuator,
+        "entity_description_kwargs": {
+            "device_class": CoverDeviceClass.WINDOW,
+            "translation_key": "attic_window_actuator",
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/cover.py:34]: You can remove the translation_key entirely.

```suggestion
```

File: custom_components/abbfreeathome_ci/cover.py [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+        },
+    },
+    "AwningActuator": {
+        "device_class": AwningActuator,
+        "entity_description_kwargs": {
+            "device_class": CoverDeviceClass.AWNING,
+            "translation_key": "awning_actuator",
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/cover.py:41]: You can remove the translation_key entirely.

```suggestion
```

File: custom_components/abbfreeathome_ci/cover.py [hunk 4]
```diff
@@ -0,0 +1,199 @@
...
+        },
+    },
+    "BlindActuator": {
+        "device_class": BlindActuator,
+        "entity_description_kwargs": {
+            "device_class": CoverDeviceClass.SHUTTER,
+            "translation_key": "blind_actuator",
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/cover.py:48]: You can remove the translation_key entirely.

```suggestion
```

File: custom_components/abbfreeathome_ci/cover.py [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+        },
+    },
+    "ShutterActuator": {
+        "device_class": ShutterActuator,
+        "entity_description_kwargs": {
+            "device_class": CoverDeviceClass.BLIND,
+            "translation_key": "cover_actuator",
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/cover.py:55]: You can remove the translation_key entirely.

```suggestion
```

File: custom_components/abbfreeathome_ci/cover.py [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+        self._device = device
+        self._sysap_serial_number = sysap_serial_number
+
+        self.entity_description = CoverEntityDescription(
+            has_entity_name=True,
+            name=device.channel_name,
+            # translation_placeholders={
+            #    "channel_id": device.channel_id,
+            #    "channel_name": device.channel_name,
+            # },
```
Comment (kingsleyadam) [inline custom_components/abbfreeathome_ci/cover.py:105]: Remove commented out code.

**Review by kingsleyadam — CHANGES_REQUESTED** [review]
Not sure if you were ready for review, but noticed these.

**Comment by derjoerg** [comment]
Ready for review :smiley:

**Comment by kingsleyadam** [comment]
> Ready for review 😃

Looks good! Will wait for a final test from @nrenner73 before merging.

**Review by kingsleyadam — APPROVED** [review]

**Pull request merged**